### DISCUSSION TITLE
i'm getting error [next-auth][warn][NO_SECRET]

### DISCUSSION THREAD

**Discussion by Ramnath0007** [nextauthjs/next-auth#8427] [Help]
### Question 💬

how should i solve this error, I have checked my environment variable and route.js, everything seems to be perfectly aligned,
I even checked it in chat gpt and other AI. I can't find a conclusion or where this error is even happening.
help me to solve this error guys.
![next auth error image](https://github.com/nextauthjs/next-auth/assets/112274891/4a9c935e-5077-45e3-ab7b-880c1887e057)

### How to reproduce ☕️

const handler = NextAuth({
  providers: [
    GoogleProvider({
      clientId: process.env.GOOGLE_ID,
      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
    })
  ],

this is my .env file in my project: 
GOOGLE_ID=************nmpmus065p0v09gfp5tg4i2r5e.apps.googleusercontent.com
GOOGLE_CLIENT_SECRET=***********************

NEXTAUTH_URL=http://localhost:3000
NEXTAUTH_URL_INTERNAL=http://localhost:3000
NEXTAUTH_SECRET=********************

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by RobinSuthar** [comment]
https://stackoverflow.com/questions/70712152/how-to-fix-no-secret-warning-thrown-by-next-auth
